lib/osf-components/addon/components/resources-list/resource-card/template.hbs

```diff
@@ -0,0 +1,16 @@
+<li local-class='resourceItem'>
+    <p local-class='resourceType'>
+        <span local-class='icon {{this.resourceIconClass}}' />
+        {{this.resourceTypeName}}
+    </p>
+    <p>
+        <OsfLink
+            @href={{@resource.pid}}
+        >
+            {{@resource.pid}}
+        </OsfLink>
+    </p>
+    <p>
+        {{@resource.description}}
```

Comment:
What about the `see more` in the case of the name and description being super long?

Reply:
Working on that now! (as well as tests for this card component)

Reply:
Oh, right, tests! Good plan. :)